### PR TITLE
Pop from stmts in the start of the loop

### PR DESCRIPTION
**Pull request by pedroarthur**

### CODE REVIEW

**Comment by axw** [comment]
@pedroarthur Thanks for the PR. Can you please describe what problem you're addressing here?

**Comment by pedroarthur** [comment]
I went into a case where `stmt[0]` (or `s`) was matching the condition at line 123 while never matching the condition at line 117. Hence, gocov got stuck and it would never return. As there is no change in the program state that would change the outcome of execution while reiterating over `stmt[0]`, I just poped it out from the slice. Unfortunately, I could not reproduce the issue in a pet project and the original source code that triggers it is proprietary.

File: gocov/convert.go [hunk 1]
```diff
@@ -113,10 +113,10 @@ func (c *converter) convertProfile(p *cover.Profile) error {
 	blocks := p.Blocks
 	for len(stmts) > 0 {
```
Comment (axw) [inline gocov/convert.go]: May as well just change this to "for _, s := range stmts {"?

Comment (pedroarthur) [inline gocov/convert.go]: Done!

**Pull request merged**